tools/triage/mainwindow.cpp

```diff
@@ -36,7 +36,7 @@ MainWindow::MainWindow(QWidget *parent) :
     mFSmodel.setFilter(QDir::Dirs | QDir::Files | QDir::NoDotAndDotDot);
     ui->directoryTree->setModel(&mFSmodel);
     QHeaderView * header =  ui->directoryTree->header();
-    for(int i = 1; i < header->length(); ++i)   // hide all except [0]
+    for (int i = 1; i < header->length(); ++i)  // hide all except [0]
```

Comment:
where does this reformatting come from.. ? From windows astyle.exe ?

Reply:
No, Linux astyle 3.0.1. These files were not formatted on Linux before.